composer.json

```diff
@@ -20,11 +20,13 @@
     "require-dev": {
         "twig/twig": "~1.22",
         "matthiasnoback/symfony-config-test": "~1.2",
-        "fabpot/php-cs-fixer": "^1.6.1",
+        "fabpot/php-cs-fixer": "^2.0",
         "satooshi/php-coveralls": "~0.6",
         "codeclimate/php-test-reporter": "dev-master",
         "symfony/phpunit-bridge": "^2.7.4"
     },
+    "minimum-stability": "dev",
```

Comment:
Sure I need it. I need it to get stable versions of package if they are available instead of dev as specified on `minimum-stability`.